### ISSUE TITLE
liquidhaskell-cabal-0.1.1.0 does not compile

### ISSUE THREAD

**Issue by peti** [spinda/liquidhaskell-cabal#10]
~~~
$ cabal new-build          
Build profile: -w ghc-8.0.2 -O1
In order, the following will be built (use -v for more details):
 - liquidhaskell-cabal-0.1.1.0 (lib) (first run)
Configuring library for liquidhaskell-cabal-0.1.1.0..
Preprocessing library for liquidhaskell-cabal-0.1.1.0..
Building library for liquidhaskell-cabal-0.1.1.0..
[1 of 1] Compiling LiquidHaskell.Cabal ( src/LiquidHaskell/Cabal.hs, /tmp/liquidhaskell-cabal-0.1.1.0/dist-newstyle/build/x86_64-linux/ghc-8.0.2/liquidhaskell-cabal-0.1.1.0/build/LiquidHaskell/Cabal.o )

src/LiquidHaskell/Cabal.hs:147:5: error:
    Not in scope: ‘ghcOptPackageKey’
    Perhaps you meant one of these:
      ‘ghcOptPackageDBs’ (imported from Distribution.Simple.Program.GHC),
      ‘ghcOptPackages’ (imported from Distribution.Simple.Program.GHC)

src/LiquidHaskell/Cabal.hs:156:5: error:
    Not in scope: ‘ghcOptSigOf’
~~~

**Comment by Wizek** [comment]
I've ran into this too. Any ideas how to overcome? Perhaps you, @spinda ?

**Comment by spinda** [comment]
Published `liquidhaskell-cabal-0.2.0.0` which should address this.
